app/controllers/notes_controller.rb

```diff
@@ -103,7 +103,11 @@ def create
         else
           flash[:notice] = "Research note published. Get the word out on <a href='/lists'>the discussion lists</a>!"
         end
-        redirect_to @node.path
+        if params[:redirect] && params[:redirect] == 'question'
```

Comment:
I see! Great to know this was considered. I'm interested in how this can be clearly communicated in how the code is implemented, and also keep path generation code centralized.

Do you think we could have `@node.path()` accept an optional parameter, like `def path(type = :default)` (best double check that for setting defaults as `:symbols` but I think it'd work) where using `@node.path(:question)` would generate the question version of the path? That also reduced redundancy in this controller.